charts/incubator/hyperswitch-app/templates/router/rollout.yaml

```
@@ -0,0 +1,169 @@
+{{- if and .Values.services.router.enabled .Values.server.rollout.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  annotations:
+    {{- with (default .Values.global.annotations .Values.server.annotations) }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  labels:
+    {{- include "hyperswitch.labels" . | nindent 4 }}
+    app.kubernetes.io/name: {{ include "hyperswitch-server.name" . }}
+    app.kubernetes.io/version: {{ .Values.services.router.version }}
+    app: {{ include "hyperswitch-server.name" . }}
+    version: {{ include "version.suffix" .Values.services.router.version }}
+  name: {{ include "hyperswitch-server.name" . }}-{{ include "version.suffix" .Values.services.router.version }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ default .Values.global.replicas .Values.server.replicas }}
+  {{- end }}
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: {{ include "hyperswitch-server.name" . }}
+      version: {{ include "version.suffix" .Values.services.router.version }}
+
+  # Argo Rollout Strategy - Canary
+  strategy:
+    canary:
+      {{- if .Values.server.rollout.canary.canaryService }}
+      canaryService: {{ include "hyperswitch-server.name" . }}-canary
+      {{- end }}
+      {{- if .Values.server.rollout.canary.stableService }}
+      stableService: {{ include "hyperswitch-server.name" . }}
+      {{- end }}
+      {{- if .Values.server.rollout.canary.trafficRouting }}
+      {{- if .Values.istio.enabled }}
+      trafficRouting:
+        istio:
+          virtualService:
+            name: {{ include "hyperswitch-server.name" . }}
+            routes:
+              - primary
+      {{- end }}
+      {{- end }}
+      steps:
+      {{- range .Values.server.rollout.canary.steps }}
+      - setWeight: {{ .setWeight }}
+      {{- if .pause }}
+      - pause:
+          {{- if .pause.duration }}
+          duration: {{ .pause.duration }}
+          {{- end }}
+      {{- end }}
+      {{- end }}
```

There are yaml format issues here
```suggestion
      {{- toYaml .Values.server.rollout.canary.steps | nindent 6 }}
```